async-nats/src/jetstream/kv/mod.rs

```diff
@@ -1250,6 +1251,7 @@ impl From<UpdateError> for CreateError {
         match error.kind() {
             UpdateErrorKind::InvalidKey => Error::from(CreateErrorKind::InvalidKey),
             UpdateErrorKind::TimedOut => Error::from(CreateErrorKind::Publish),
+            UpdateErrorKind::WrongLastSequence => Error::from(CreateErrorKind::AlreadyExists),
```

Comment:
We should not expose Stream internals here, so for consistency, I would rename it.

Maybe `WrongLastRevision`?

Reply:
Ah yes, this makes a lot of sense (the sequence number is the underlying impl for the revision but that should be opaque to the K/V user), I've updated PR commit with a rename of the variant to `WrongLastRevision`.

We were trying to game out in this scenario whether the caller of `kv.create()` would care to know if this situation occurred, but it seemed like simply knowing that the caller didn't successfully create the key themselves (and that another writer may have slipped in an created the key) was enough.

Thanks again for such a quick review!